src/commands/sdk/query.ts

```diff
@@ -99,10 +111,14 @@ Other
 
     const response = await this.sdk?.query(request)
 
-    this.log(JSON.stringify(response, null, 2))
+    const display = this.flags.display === ''
+      ? response
+      : _.get(response, this.flags.display)
```

Comment:
it's usualy better pratice to import lodash as so `import get from 'lodash/get`

Reply:
It's because the size of imported code is smaller?

Reply:
In some files, I need to import many methods from lodash `isEmpty`, `isPlainObject`, etc
I don't like it much to have 5 lines of import instead of one :(

Reply:
The overall importe size is far lower indeed :)
<img width="394" alt="Screenshot 2020-05-15 at 09 42 58" src="https://user-images.githubusercontent.com/10114634/82024712-7b851e00-9690-11ea-8e2c-16ab7eb3fe05.png">

Reply:
I agree but since this is not a web application people have to download lodash entirely so it won't change the download time.
But it's still a good point, I'm gonna add every lodash modules separately to reduce the total size

Reply:
EDIT: since `lodash` is already a dependency of Kourou, even if I use native methods we won't gain any size..

Reply:
Also, required modules are cached, so if the same module is required in multiple files, it doesn't change neither the memory consumption nor the time taken to require the code.

Reply:
That's true server side indeed, usually a good practice client side :)